docs/CHANGELOG.md

```diff
@@ -4,6 +4,9 @@
 
 ### Breaking
 
+## 3.0.3
+* Add ubuntu-22.10 Dockerfile **[@OctopusET]**
```

Comment:
This PR is neither breaking change nor 3.0.3.

Reply:
fixed